django/db/backends/postgresql/schema.py

```diff
@@ -15,11 +15,11 @@ class DatabaseSchemaEditor(BaseDatabaseSchemaEditor):
     sql_delete_sequence = "DROP SEQUENCE IF EXISTS %(sequence)s CASCADE"
 
     sql_create_index = (
-        "CREATE INDEX %(name)s ON %(table)s%(using)s "
+        "CREATE INDEX IF NOT EXISTS %(name)s ON %(table)s%(using)s "
         "(%(columns)s)%(include)s%(extra)s%(condition)s"
     )
     sql_create_index_concurrently = (
-        "CREATE INDEX CONCURRENTLY %(name)s ON %(table)s%(using)s "
+        "CREATE INDEX CONCURRENTLY IF NOT EXISTS %(name)s ON %(table)s%(using)s "
```

Comment:
I think it is unacceptable, because it will hide real issues from users. Users should be inform when an index already exists, as it may be a different constraint then they expect. This change is also backward incompatible as it changes the current behavior for creating indexes.

Reply:
OK, thanks @felixxm. I'll revert those changes.